index.html

```diff
@@ -0,0 +1,180 @@
+                            <a href="#courses" class="link">Courses</a>
+                        </li>
+                        <li class="nav_items" onclick="closeMobileMenu()">
+                            <a href="#lecturers" class="link">Lecturers</a>
+                        </li>
+                        <li class="nav_items" onclick="closeMobileMenu()" >
+                            <a href="#partners" class="link">Partners</a>
+                        </li>
+                        <li class="nav_items" onclick="closeMobileMenu()">
+                            <a href="#" class="linkB">OW Inno.</a>
+                        </li>
+                    </ul>
+                </nav>
+                <div class="nav_wrapper">
+                    <nav class="desktop_nav">
+                        <div class="nav-logo">
+                            <a href="./index.html"><img src="images/logo-removebg-preview.png" class="nav-logo" alt="logo of Orbit Web Innovations"></a>
+                        </div>
+                        <ul class="nav_wrapper_desk">
+                            <li class="nav_items_desk">
+                                <a href="./about.html" class="link_deskA">About</a> 
+                            </li>
+                            <li class="nav_items_desk">
+                                <a href="#courses" class="link_desk">Program</a>
+                            </li>
+                            <li class="nav_items_desk">
+                                <a href="#courses" class="link_desk">Courses</a>
+                            </li>
+                            <li class="nav_items_desk">
+                                <a href="#lecturers" class="link_desk">Lecturers</a>
+                            </li>
+                            <li class="nav_items_desk">
+                                <a href="#partners" class="link_desk">Partners</a>
+                            </li>
+                            <li class="nav_items_desk">
+                                <a href="#" class="link_deskB">OW Inno.</a>
+                            </li>
+                        </ul>
+                    </nav>
+                </div>
+        </div>
+    </header>
+    <section class="hero">
+                <div class="hero_box">
+                    <h1 class="hero_heading">"Hello! Sharing World"</h1>
+                    <h2 class="hero_title">ORBIT WEB INNOVATIONS</h2>
+                </div>
+                <div class="hero_description">
+                    <p class="des1">At Orbit institute we believe in nurturing great minds to become web programming professionals through our tailor-made courses. </p>
+                </div>
+                    <p class="des2">
+                        COURSE-TIME: (MON) ~ (FRI)
+                    </p>
+                    <p class="des3">
+                        @ Orbit Web Innovations. Oslo, Norway.
+                    </p>
```

Comment:
- [ ] Kindly check the `font-weight` and `font-size` to match the design. 🧐️

| Your design                                                             | Template                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/217485711-54e39c59-e2b9-4924-869e-83917d13301f.png" width="400">|<img src="https://user-images.githubusercontent.com/17243325/217485703-e345524c-c95e-4dc7-a90a-c37262e6b134.png" width="400">|